components/Link.js

```diff
@@ -47,17 +50,9 @@ class Link extends React.Component {
           {children}
         </Scrollchor>
       );
-    } else if (this.isIframe) {
-      const routeFromRouter = router.findByName(route);
-      const path = routeFromRouter ? routeFromRouter.getAs(params) : `https://opencollective.com${route}`;
-      return (
-        <a href={path} title={title} target="_top" className={className} {...otherProps}>
-          {children}
-        </a>
-      );
```

Comment:
@Betree : Sorry, but I am not sure what you mean by widgets here? So these widgets are not part of our tests? 🤔

Reply:
The ones that projects integrate into their README or on their websites. You can preview some of them on https://opencollective.com/widgets. Unfortunately, these are based on old code and have very poor test coverage. I think the main one is `banner.html`

Reply:
You can see one here: https://kivy.org/#home. If you try to manually change the iframe URL to https://opencollective-frontend-bj8twodx4.vercel.app/kivy/banner.html?limit=10&id=banner-iframe-6108&style={} you'll see that it indeed creates a regression: the page should open in a new tab. So I guess we should keep this for now.

![Peek 2021-02-15 18-44](https://user-images.githubusercontent.com/1556356/107978891-d8b25500-6fbd-11eb-9e3f-45275bf85a71.gif)

Reply:
@Betree : I tested that. But even without the url change it doesn't seem to open in a new tab anyways. So I guess that is not related to this change? 🤔 

![pre-test](https://user-images.githubusercontent.com/12435965/107985216-80a82e00-6f7e-11eb-84f6-a00c64b8f6ae.gif)

Reply:
You're actually right, I'll open a separate issue about that. Since this code is not playing its role, we can indeed remove it.

Reply:
Issue created here: https://github.com/opencollective/opencollective/issues/3984

If you see a quick fix, feel free to implement it in this PR. Otherwise, we can keep that for later

Reply:
@Betree : Took a quick look and I couldn't figure something obvious so we'll leave it for the other issue. 🤔